scripts/test-acceptance.sh

```diff
@@ -32,6 +32,7 @@ mkdir -p test/logs
 cd test/logs
 
 test_name="${1:-acceptance}"
+test=":MultipleOpenAssessmentTest"
```

Comment:
After getting thumbs-ups, I will un-check this file and squash my commits. This just for a fast jenkins run.